docs/index.md

```diff
@@ -1,6 +1,6 @@
 # dbt_meshify
 
-`dbt-meshify` is a dbt-core plugin that automates the management and creation of dbt-core model governance features introduced in dbt-core v1.5. Each command in the package will leverage your dbt project metadata to create and/or edit the files in your project to properly configure the models in your project with these governance features.
+`dbt-meshify` is a dbt-core plugin that automates the creation of dbt-core model governance features introduced in dbt-core v1.5. This package will leverage your dbt project metadata to create and/or edit the files in your project to properly configure the models in your project with these governance features.
```

Comment:
question for the group -- is this a "plugin"? I am genuinely unsure of the definition of plugin, and whether or not this cli, which makes use of the dbt's programmatic invocations actually counts as something that plugs into dbt core or just rides along with it

Reply:
i think we should just call it a CLI tool

Reply:
I also believe that this should be referred to as a CLI tool or utility. IMHO, dbt Core plugins would specifically refer to programs that are "plugged into" a dbt Core context, as opposed to tools that create a dbt Core context to perform operations. In a plugin scenario, dbt Core is the operator. In our scenario, `dbt-meshify` is the operator.

Reply:
ok - will update this!